app/controllers/concerns/multi_file_uploadable.rb

```diff
@@ -0,0 +1,79 @@
+module MultiFileUploadable
+  extend ActiveSupport::Concern
+
+  included do
+    skip_before_action :verify_authenticity_token, only: [:create, :destroy]
+  end
+
+  def create
+    unless supported_filetype(params[:documents])
+      return return_error(nil,
+                          { message: t('activemodel.errors.messages.forbidden_document_type') })
+    end
+
+    evidence = upload_file(params)
+    return_success({ evidence_id: evidence.id, file_name: params[:documents].original_filename })
+  rescue FileUpload::FileUploader::PotentialMalwareError => e
+    return_error(e, { message: t('activemodel.errors.messages.suspected_malware') })
+  rescue StandardError => e
+    return_error(e, { message: t('activemodel.errors.messages.upload_failed') })
+  end
+
+  # :nocov:
+  def destroy
+    raise 'Implement in subclass'
```

Comment:
I don't think there's much value in these. First, this is a module, so there are no subclasses. Second, these aren't methods that get called by other methods in this module, so these raises will never get triggered by someone forgetting to implement them. I'd just delete them.